components/rollup_node/README.md

```diff
@@ -1,62 +1,109 @@
 # Rollup Node
 
-The consensus module of Optimistic Ethereum.
+The consensus-layer of Optimism.
+
+This design mirrors the separation of L1 consensus- and execution-layer to L2,
+for maximum compatibility and minimal complexity.
 
 ## Summary
 
-The Rollup Node is a consensus client that determines the latest state of the rollup. It reads state from L1, and possibly the sequencer, to compute the L2 state. The [block generation spec][block-gen] defines the rules by which L1 state is converted into L2 blocks.
+The Rollup Node is a consensus client that tracks the rollup:
+- Fork-choice: following L1
+- Input Data: deposits and sequencer data extracted from L1
+- Latest blocks: optimistically via L2 p2p, overrideable by L1
+
+The input data and forkchoice information collected from L1 (and optimistically via p2p)
```

Comment:
>  optimically because we don't wait for L1 confirmations?

Exactly

> "via p2p": as-is it's more confusing than nothing - should expand or remove

Again, this is the shortest summary on top of readme doc. This is not the place to go in-depth about p2p stack. And yes, that's exactly what it means: use the p2p interface to float around these blocks. I'll change it to "propagate optimistically via p2p interface", hope that helps.